azure-pipelines.yml

```diff
@@ -10,13 +10,18 @@ pr:
 jobs:
 - job: 'Test'
   pool:
-    vmImage: 'ubuntu-latest'
-
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '3.6.9'
+      Python37:
+        python.version: '3.7.4'
   steps:
   - task: UsePythonVersion@0
     inputs:
-      versionSpec: '3.7'
-    displayName: 'Set python version to 3.7'
+      versionSpec: '${python.version}'
```

Comment:
Bottom right in the Eisenhower matrix, but this might work if you use `$()` instead of `${}` as seen [here](https://docs.microsoft.com/en-us/azure/devops/pipelines/ecosystems/python?view=azure-devops).